README.md

```diff
@@ -1 +1,44 @@
-# RESTAURANT-PAGE
\ No newline at end of file
+# Restaurant Page
+
+## Description
+
+This is a JavaScript project from Odin's curriculum. It is based on DOM manipulation by dynamically rendering a simple restaurant homepage.
+There I used JavaScript alone to generate the entire contents of the website.
+
+## Built With
+
+- JavaScript
+- Webpack
+
+## Set up this projecect locally
+
+- Open the terminal in the directory where you want to install the app;
+- Run command `git clone git@github.com:FrederickMih/RESTAURANT-PAGE.git` which will clone the project;
+- Switch to the main directory of project using cd;
+- Run `npm install webpack webpack-cli --save-dev` in your computer terminal to install Webpack (you will need Node.js installed on your machine);
```

Comment:
- [ ] [Optional] Since running this project, we simply open the `index.html` file found in `dist/` folder, that means you pushed a built version, so unless we need to build it ourselves locally, we don't need to install these dependencies.